db/school.sql

```diff
@@ -75,15 +81,44 @@ CREATE TABLE `registrar` (
 --
 
 CREATE TABLE `student` (
-                           `id` int(10) UNSIGNED NOT NULL,
-                           `first_name` varchar(75) NOT NULL,
-                           `last_name` varchar(75) NOT NULL,
-                           `address` varchar(100) NOT NULL,
-                           `email` varchar(100) NOT NULL,
-                           `phone_number` varchar(50) NOT NULL,
-                           `date_of_birth` date NOT NULL
+  `id` int(10) UNSIGNED NOT NULL,
+  `first_name` varchar(75) NOT NULL,
+  `last_name` varchar(75) NOT NULL,
+  `address` varchar(100) NOT NULL,
+  `email` varchar(100) NOT NULL,
+  `phone_number` varchar(50) NOT NULL,
+  `date_of_birth` date NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+--
+-- Dumping data for table `student`
+--
+
+INSERT INTO `student` (`id`, `first_name`, `last_name`, `address`, `email`, `phone_number`, `date_of_birth`) VALUES
+(98765432, 'Test', 'Student', 'Test Student Street', 'happygero7@gmail.com', '222-333-4444', '1648-08-14');
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `useraccounts`
+--
+
+CREATE TABLE `useraccounts` (
```

Comment:
What's usually the standard for table names with more than one words?  An underscore in between maybe?  Also you could just make it a table name of one word (ie. user).

Reply:
that is true, but that would require refactoring the files and I don't feel like it :) especially considering that it is a pretty minor change

Reply:
Alright, minor let's merge this